### ISSUE TITLE
Pressing 'Enter' doesn't work on 'Rename' dialog for ADLS Gen2 accounts

### ISSUE THREAD

**Issue by v-xuanzh** [Microsoft/AzureStorageExplorer#873]
Storage Explorer Version: 1.5.0
Platform/OS Version: Windows 10/ MacOS High Sierra/ Linux Ubuntu 16.04
Architecture: ia32
Build Number: 20181117.12
Commit: 78a0d1ab
Regression From: Not a regression

#### Steps to Reproduce: ####
1. Create one account with ADLS Gen2 enabled.
2. Create a blob container under the account -> Upload a file to it.
3. Right click the uploaded file -> Select 'Rename' -> Type a valid file name then try to press 'Enter' directly.

#### Expected Experience: ####
The file can be renamed successfully.

#### Actual Experience: ####
Pressing 'Enter' doesn't work.
